app/assets/javascripts/discourse/app/components/user-nav/dropdown-list.hbs

```diff
@@ -0,0 +1,18 @@
+<li class={{concat @class " user-nav-dropdown-list-item"}}>
+  <button type="button" class={{this.buttonClass}} {{on "click" this.toggleList}}>
+    {{d-icon @icon}}
+    <span>{{@text}}</span>
+    {{d-icon this.chevron class="user-nav-dropdown-chevron"}}
+  </button>
+
+  {{#if (and (has-block "submenu") this.displayList)}}
```

Comment:
I guess this relates to `:submenu` somehow, so it only yields if that is shown on the page (e.g. if the user profile is not hidden?)

Reply:
Some of the other navigation links will not have a dropdown list so I'm adding this conditional in preparation for future work :grin: Sorry I should have left a comment to explain why I'm doing this.